README.md

```diff
@@ -1 +1,47 @@
-# Kamran123Ahmad.github.io
\ No newline at end of file
+# Kamran123Ahmad.github.io
+# Project Name
+> This project is a portfolio website which is created using pure Html5 and Css3.
+
+## Built With
+- Html
+- Css
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+Git must be installed on you machine.
+
+### Setup
+Clone this repository by using the command line :
+`git clone https://github.com/Kamran123Ahmad/My_Portfolio_Project`
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [Kamran123Ahmad](https://github.com/Kamran123Ahmad)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.
```

Comment:
- [ ]  Kyndly add the link of your deployed page on your readme file 

![image](https://user-images.githubusercontent.com/95243996/200337373-f21b5bcd-2b1a-4c35-bce2-7abc4ea3a660.png)
![image](https://user-images.githubusercontent.com/95243996/200337449-2f27bda4-b18b-43cd-bcd4-3d61aec95152.png)